Release 1.1.0: release -> master

TB-1993 opened the pull request.

Commit:
Merge branch 'feature/gh1_initial_creation_of_xts' into develop

Commit:
Merge branch 'release/1.0.0'

Commit:
Merge tag '1.0.0' into develop

1.0.0

TB-1993 committed:
Update #6: Initial creation of xts server client

Commit:
added functionality to send rest api requests for allocation, deallocation, search, and listing slots

Commit:
add test for xts allocator client

zghp committed:
clean up imports

Commit:
add --help/-h args, clean up imports and remove extra comments

Commit:
Merge branch 'feature/gh6_create_xts_allocator_client' of https://github.com/rdkcentral/xts_core into feature/gh6_create_xts_allocator_client

Commit:
clean up allocate_slot() and get_command_choices() methods and add docstrings

Commit:
remove comments

zghp committed:
Merge pull request #8 from rdkcentral/feature/gh6_create_xts_allocator_client

gh6 - create xts allocator client

TB-1993 committed:
Bumped changelog

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/xts_core?pullRequest=13) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/xts_core?pullRequest=13) before we can accept your contribution.<br/><hr/>**TB-1993** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/xts_core?pullRequest=13) it.</sub>

TB-1993 committed:
Initial commit

TB-1993 committed:
gh1 - Initial creation of xts script

TB-1993 committed:
Add #1: Added installer for xts

TB-1993 committed:
Update #1: Updated README.md with information about xts

TB-1993 committed:
Bug #1: fixed a couple of typos

TB-1993 committed:
Update #1: Updated requirements to pull the released version of
yaml_runner

TB-1993 committed:
Fix #1: Fixes based on review comments

Correct typos in README. Removed debug code from xty.py. Added comment
to explain output redirect in install.sh

TB-1993 committed:
Release 1.0.0: Updated changelog
